sw/host/spiflash/spiflash.cc

```diff
@@ -160,12 +160,14 @@ bool ParseArgs(int argc, char **argv, SpiFlashOpts *options) {
   assert(options);
   const struct option long_options[] = {
       {"input", required_argument, nullptr, 'i'},
-      {"dev-id", required_argument, nullptr, 'd'},
-      {"dev-sn", required_argument, nullptr, 'n'},
-      {"dump-frames", required_argument, nullptr, 'x'},
-      {"verilator", required_argument, nullptr, 's'},
+      {"dev-id", optional_argument, nullptr, 'd'},
+      {"dev-sn", optional_argument, nullptr, 'n'},
+      {"dump-frames", optional_argument, nullptr, 'x'},
+      {"verilator", optional_argument, nullptr, 's'},
```

Comment:
These should still be marked as `required_argument` as we need a value associated with each arg.

Reply:
Agree. read the manpage too quickly there and thought this was about the need for the arg not if it had a value. Please also see other comment.

Reply:
Fixed in latest.